src/Components/Search/Search.js

```diff
@@ -5,7 +5,7 @@ import searchIcon from '../../Assets/search.png';
 
 const Search = ({addRecipes}) => {
 
-  // const [input]
+  const [inputError, setInputError] = useState('Please enter valid search')
```

Comment:
This is also another great message for the user that is increasing the user interface.